src/System.Data.SqlClient/src/System/Data/SqlClient/SqlErrorCollection.cs

```diff
@@ -2,64 +2,36 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-
-
-//------------------------------------------------------------------------------
-
 using System.Collections;
-
+using System.Collections.Generic;
 
 namespace System.Data.SqlClient
 {
     public sealed class SqlErrorCollection : ICollection
```

Comment:
Maybe this should implement `ICollection<SqlError>` instead of `ICollection`? I see `CopyTo(SqlError[])` and `Add(SqlError)` methods are present.